core/src/avm2/globals/flash/system/capabilities.rs

```diff
@@ -3,16 +3,50 @@
 use crate::avm2::{Activation, AvmString, Error, Object, Value};
 use crate::player::PlayerRuntime;
 
+/// Implements `flash.system.Capabilities.os`
+pub fn get_os<'gc>(
+    activation: &mut Activation<'_, 'gc>,
+    _this: Object<'gc>,
+    _args: &[Value<'gc>],
+) -> Result<Value<'gc>, Error<'gc>> {
+    let os = match activation.avm2().player_runtime {
+        // For most normal Flash Player usage, the OS should not matter,
+        // so let's pretend it's Windows for the broadest possible compatibility.
+        PlayerRuntime::FlashPlayer => "Windows 8",
+        PlayerRuntime::AIR => {
+            if cfg!(windows) {
+                "Windows 10"
```

Comment:
Why windows 10 here but windows 8 above? :D

not a blocker, just curious

Reply:
No real reason. I just kept Windows 8 because we were already using it.

Reply:
Yeah 8 as the general default is smart (I'm fairly sure some swfs are subject to the windows 9 problem lol)

Reply:
Then why report Windows 10 to AIR? It's a lie anyway...

Reply:
Good question. I just took that from #17517. It's possible that most AIR content might be newer than Flash in general and is more likely to be running on Windows 10? On the other hand wikipedia claims AIR was first released in 2008, which is much earlier than I had expected.

Reply:
I suppose it doesn't matter anyway. If anyone ever discovers that it causes problems to any content, we can fix it then.